lib/oper1.g

```diff
@@ -227,8 +232,17 @@ BIND_GLOBAL( "INSTALL_METHOD_FLAGS",
               ": <method> must be a function, `true', or `false'" );
     fi;
     methods[i+(narg+3)] := rank;
-
     methods[i+(narg+4)] := IMMUTABLE_COPY_OBJ(info);
+    # This is a horrible hack because I don't want to change the length
+    # of METHODS_OPERATION at the moment
+    ADD_LIST( METHOD_LOCATIONS,
+              [ opr,
+                method,
+                narg,
+                flags,
+                rank,
+                rec( line := INPUT_LINENUMBER(), file := INPUT_FILENAME() ) ] );
```

Comment:
So what does this add compared to `FILENAME_FUNC(method)`? The location the method is installed? So that would differ for the few instances were a function is defined in one place, and then installed in another? That seems somewhat rare... But in any case: If I guess right, then this should be explicitly documented, instead of leaving the reader to wonder.